cypress/integration/chi-custom-elements/drawer.spec.js

```diff
@@ -55,6 +54,19 @@ describe('Drawer', function() {
 
   });
 
+  it('Should have a title if attribute title="" is provided ', function() {
+
+    cy.get('[data-cy="test-top-with-title"]')
+      .scrollIntoView()
```

Comment:
ScrollIntoView might be unnecessary as Cypress automatically do this with default configuration.